src/pages/resources/recipes/persistent-storage-migration/index.md

```diff
@@ -0,0 +1,79 @@
+---
+keywords:
+  - API Documentation
+  - UXP
+  - Plugin Storage
+  - Persistent Storage
+  - Scripting
+title: Persistent Storage Migration in InDesign UXP
+
+---
+
+# Persistent Storage Migration
+Persistent storage migration in InDesign allows users to seamlessly carry over plugin data from one version to the next. By selecting the 'Import Previous Settings and Preferences' option during installation, users can ensure that their plugin data is preserved when upgrading to the latest version of InDesign.
+
+
+<InlineAlert variant="info" slots="header, text1, text2"/>
+
+Plugins and Scripts
+
+**In plugins**, Supported.
+
+**In scripts**, Not Supported.
+
+
+
+## System requirements
+Please make sure your local environment uses the following application versions before proceeding.
+- InDesign v20.0 or higher
+
+## Import Previous Settings and Preferences
+
+![CCD Setting](./CCD-setting.png)
+
+While installing the new InDesign version, if the "Import Previous Settings and Preferences" option is selected in the Creative Cloud Desktop (CCD) app, the plugin data would be copied to the following folder locations:
+
+
+**Plugins Installed from CCD or Plugin Market Place (External)**
+
+- **Windows:**
+  *AppData\Roaming\Adobe\UXP\PluginsStorage\IDSN\\{App Version}\External*
```

Comment:
A minor comment, use back ticks for small code snippets and paths. Its easy to copy.
`AppData\Roaming\Adobe\UXP\PluginsStorage\IDSN\\{App Version}\External`